spec/views/users/webauthn_setup/new.html.erb_spec.rb

```diff
@@ -32,11 +32,43 @@
 
       render
     end
+    context 'when user selects multiple MFA options on account creation' do
+      before do
+        allow(view).to receive(:in_multi_mfa_selection_flow?).and_return(true)
+        assign(:need_to_set_up_additional_mfa, false)
+      end
 
-    it 'displays warning alert' do
-      render
+      it 'does not displays info alert' do
+        render
+
+        expect(rendered).to_not have_content(I18n.t('forms.webauthn_platform_setup.info_text'))
+      end
+    end
+
+    context 'when user selects only platform auth options on account creation' do
+      before do
+        allow(view).to receive(:in_multi_mfa_selection_flow?).and_return(true)
```

Comment:
I don't think this is actually doing anything, since the view is unaware of `in_multi_mfa_selection_flow?`. I might expect there to be some controller specs which test how the value of `need_to_set_up_additional_mfa` is assigned, where we could consider this kind of stubbing. I think it's fine for the view specs to assign the value like what's done on the line following this.

Reply:
I'd agree, I think it would be useful to add controller specs on top of the existing ones.

Reply:
@mdiarra3 Let's add some test coverage to `spec/controllers/users/webauthn_setup_controller_spec.rb` to verify that `@need_to_set_up_additional_mfa` is assigned correctly based on different combinations of `@platform_authenticator`, `in_multi_mfa_selection_flow?`, and `mfa_selection_count`.